enterprise/dev/ci/internal/ci/operations.go

```diff
@@ -257,6 +257,7 @@ func addBrowserExtensionIntegrationTests(parallelTestCount int) operations.Opera
 				bk.Cmd("yarn run cover-browser-integration"),
 				bk.Cmd("yarn nyc report -r json"),
 				bk.Cmd("dev/ci/codecov.sh -c -F typescript -F integration"),
+				bk.AutomaticRetry(2), // Temporary
```

Comment:
```suggestion
				bk.AutomaticRetry(1), // Temporary
```

if one retry doesn't work we should start thinking about disabling 😬